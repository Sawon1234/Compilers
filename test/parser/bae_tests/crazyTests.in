fun fourAmMadness ( Trolololo lolololo) : LiChiXi
{
   return "oh hai";
}

fun Xiouyue(Little moon) : Guo
{
	return("Hi Mandy");
}

fun is(A key) : Word
{
	return yeahYEAH;
}

fun wtfAmIDoing(I dont) : Know 
{
	return asdfasdfasdf;
}

 fun foo(y : Integer) : Integer return y * 2; 

 fun foo(y : Integer) : Integer = y * 2;
