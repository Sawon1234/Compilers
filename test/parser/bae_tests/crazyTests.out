fun fourAmMadness ( lolololo : Trolololo < > ) : LiChiXi { return "oh hai" ; } fun Xiaoyue ( moon : Little < > ) : Guo {	return ( "Hi Mandy" ) ; } fun is ( key : A < > ) : Word { return yeahYEAH; } fun wtfAmIDoing ( dont : I < > ) : Know { return asdfasdfasdf ; } fun foo ( y : Integer < > ) : Integer return y * 2; fun foo( y : Integer < > ) : Integer = y * 2 ;
